src/style.css

```diff
@@ -1 +1,66 @@
 /* styles go here */
+.form_field {
+  display: block;
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+.fieldset-1 {
```

Comment:
Consider using a more descriptive class name than '.fieldset-1'. It should be reflective of its purpose or the content it styles.